notebooks/2.modeling/nbconverted/2.modeling.py

```diff
@@ -182,14 +212,15 @@
 
 # ### Evaluating Multi-class model trained with original split with holdout data
 
-# In[ ]:
+# In[12]:
 
 
 # evaluating with plate holdout
 plate_ho_precision_recall_df, plate_ho_f1_score_df = evaluate(
     model=best_model,
```

Comment:
I noticed these X data sets are not shuffled?

Reply:
Hey @MikeLippincott! the shuffled data is shown [here](https://github.com/WayScience/Cytotoxic-Nuisance-Metadata-Analysis/blob/d3984e413cb9127db95c8a69de8ee876bc87ed33/notebooks/2.modeling/nbconverted/2.modeling.py#L143-L153)